introducing-java/src/test/java/com/onto/javacourse/intro/summingLargeNumbers/SummingLargeNumbersTest.java

```diff
@@ -0,0 +1,16 @@
+import org.junit.jupiter.api.Test;
+
+import java.util.*;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class SummingLargeNumbersTest {
+
+    @Test
+    public void sumNumberTest() {
```

Comment:
You should try to think of better test cases. What if I pass as parameters "000001" + "9"? What about "999999999999" + "1" and etc. Also, what will happen if I pass as input "asd" + "1"?